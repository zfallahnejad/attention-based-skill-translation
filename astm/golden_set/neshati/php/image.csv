187606,image
118068,image
1226894,image
476,image
507674,image
367456,image
42139,image
138475,image
362536,image
541091,image
1415724,image
3775731,image
345031,image
250259,image
889949,image
350858,image
512922,image
