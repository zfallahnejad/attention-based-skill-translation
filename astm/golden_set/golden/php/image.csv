187606,44,42,0.5116279069767442
118068,40,54,0.425531914893617
1226894,24,15,0.6153846153846154
476,17,12,0.5862068965517241
507674,15,18,0.45454545454545453
367456,14,18,0.4375
42139,13,19,0.40625
362536,13,9,0.5909090909090909
138475,13,9,0.5909090909090909
541091,9,7,0.5625
345031,8,8,0.5
3775731,8,5,0.6153846153846154
1415724,8,4,0.6666666666666666
350858,7,5,0.5833333333333334
512922,7,4,0.6363636363636364
889949,7,6,0.5384615384615384
250259,7,4,0.6363636363636364
